src/azure-cli/azure/cli/command_modules/storage/operations/account.py

```diff
@@ -443,3 +443,36 @@ def update_blob_service_properties(cmd, instance, enable_change_feed=None, enabl
         instance.restore_policy = cmd.get_models('RestorePolicyProperties')(
             enabled=enable_restore_policy, days=restore_days)
     return instance
+
+
+def create_encryption_scope(cmd, client, resource_group_name, account_name, encryption_scope_name,
+                            encryption_key_source=None, encryption_key_uri=None):
+    EncryptionScope = cmd.get_models('EncryptionScope')
+
+    if encryption_key_source:
+        encryption_scope = EncryptionScope(source=encryption_key_source)
+
+    if encryption_key_uri:
+        EncryptionScopeKeyVaultProperties = cmd.get_models('EncryptionScopeKeyVaultProperties')
+        encryption_scope.key_vault_properties = EncryptionScopeKeyVaultProperties(key_uri=encryption_key_uri)
```

Comment:
When the `encryption_key_uri` has a value but the `encryption_key_source` has no value, the `encryption_scope` will not be assigned, will it cause the null pointer exception of the `encryption_scope.key_vault_properties`?

Reply:
Actually it will key uri will be ignored by server side. But it is a good point to check key uri for key source.